crates/bevy_render/src/texture/mod.rs

```diff
@@ -170,8 +174,176 @@ pub trait BevyDefault {
+    R8Unorm,
+
+    /// Red and green channels. 8 bit integer per channel. [0, 255] converted to/from float [0, 1] in shader.
+    Rg8Unorm,
+
+    /// Red, green, blue, and alpha channels. 8 bit integer per channel. [0, 255] converted to/from float [0, 1] in shader.
+    Rgba8Unorm,
+
+    /// The default texture format used by bevy. It being clamped means view targets using it won't support effects like bloom.
+    /// Red, green, blue, and alpha channels. 8 bit integer per channel. Srgb-color [0, 255] converted to/from linear-color float [0, 1] in shader.
+    #[default]
+    Rgba8UnormSrgb,
+
+    /// Blue, green, red, and alpha channels. 8 bit integer per channel. [0, 255] converted to/from float [0, 1] in shader.
+    Bgra8Unorm,
+
+    /// Blue, green, red, and alpha channels. 8 bit integer per channel. Srgb-color [0, 255] converted to/from linear-color float [0, 1] in shader.
+    Bgra8UnormSrgb,
+
+    /// Red channel only. 16 bit float per channel. Float in shader.
+    R16Float,
+
+    /// Red and green channels. 16 bit float per channel. Float in shader.
+    Rg16Float,
+
+    // BEFORE_MERGE: Figure these out. Since they're native only they won't be in the table.
+    //
+    // R16Unorm
+    // R16Snorm
+    // Rg16Unorm
+    // Rg16Snorm
+    // Rgba16Unorm
+    // Rgba16Snorm
+    /// A common texture format for color in HDR textures and the default texture format used by bevy when it needs unclamped texture format.
+    ///
+    /// Red, green, blue, and alpha channels. 16 bit float per channel. Float in shader.
+    Rgba16Float,
+
+    /// A cheaper to render unclamped texture format than [`ViewTargetFormat::UNCLAMPED_DEFAULT`]. Might encounter to precision issues.
+    ///
+    /// Can only be used if the render device supports the [`RG11B10UFLOAT_RENDERABLE`](wgpu::Features::RG11B10UFLOAT_RENDERABLE)
+    /// feature.
+    ///
+    /// Red, green, and blue channels. 11 bit float with no sign bit for RG channels. 10 bit float with no sign bit for blue channel. Float in shader.
+    Rb11b10Float,
+
+    /// Red, green, blue, and alpha channels. 10 bit integer for RGB channels, 2 bit integer for alpha channel. [0, 1023] ([0, 3] for alpha) converted to/from float [0, 1] in shader.
+    Rgb10a2Unorm,
+    // TODO: These may be supported in the future. See https://github.com/gpuweb/gpuweb/issues/3556.
+    //
+    // If/When this happen they'll require the `float32-blendable` (pressumed name) + `float32-filterable`
+    //
+    // /// Red channel only. 32 bit float per channel. Float in shader.
+    // R32Float,
+    // /// Red and green channels. 32 bit float per channel. Float in shader.
+    // Rg32Float,
+    // /// Red, green, blue, and alpha channels. 32 bit float per channel. Float in shader.
+    // Rgba32Float,
+}
+
+impl ViewTargetFormat {
+    pub const DEFAULT: Self = Self::Rgba8UnormSrgb;
+    pub const UNCLAMPED_DEFAULT: Self = Self::Rgba16Float;
```

Comment:
I don't mind the unclamped name for the function but I think this may be more confusing as a public default for users.